elastic/apm-server
# Update `go-docappender` dependency to latest

## Motivation/summary

Includes a fix to remove the contents of a field that failed to index due to mapping issues.

## Checklist


## How to test these changes

1. Update a field mapping to some incorrect value
2. Send data
3. Verify that even though a mapper exception is logged, no "Preview" of the field is logged.
